cli/tests/test_completion.rs

```diff
@@ -415,21 +425,14 @@ fn test_revisions() {
     k	working_copy
     y	mutable
     zq	remote_commit
+    alias_with_newline	    roots(
+    siblings	@-+ ~@
     ");
 
     // complete args of the default command
     test_env.add_config("ui.default-command = 'log'");
     let stdout = test_env.jj_cmd_success(&repo_path, &["--", "jj", "-r", ""]);
-    insta::assert_snapshot!(stdout, @r"
-    immutable_bookmark	immutable
-    mutable_bookmark	mutable
-    k	working_copy
-    y	mutable
-    q	immutable
-    zq	remote_commit
-    zz	(no description set)
-    remote_bookmark@origin	remote_commit
-    ");
+    insta::assert_snapshot!(stdout, @"immutable_bookmark\timmutable\nmutable_bookmark\tmutable\nk\tworking_copy\ny\tmutable\nq\timmutable\nzq\tremote_commit\nzz\t(no description set)\nremote_bookmark@origin\tremote_commit\nalias_with_newline\t    roots(\nsiblings\t@-+ ~@");
```

Comment:
I updated insta, and now it seems to prefer outputting '\n' over literal newlines. Can't say I'm a fan.

Reply:
Oh, that's weird. Maybe file a bug report?

Reply:
https://github.com/mitsuhiko/insta/issues/715